app/helpers/users_helper.rb

```diff
@@ -1,9 +1,6 @@
 module UsersHelper
   def gravatar(user = nil, size = 154)
-    default_image = URI.encode(image_url('user.png'))
-
-    image_path = "#{user&.gravatar_path}?s=#{size}"
-    image_path = image_path + "&d=#{default_image}" unless Rails.env == 'development'
+    image_path = "#{user&.gravatar_path}?s=#{size}&d=mp"
```

Comment:
Apparently just adding "d=mp" provides a "missing person" default avatar from gravatar app. No need to host this asset ourselves.